functionality/pub.py

```diff
@@ -0,0 +1,58 @@
+import paho.mqtt.client as mqtt
+import serial
+import json
+
+'''
+main methods
+- connect()
+- disconnect()
+- subscribe()
+- publish ()
+'''
+
+class Publisher:
+    pub = ""
+    broker_address = ""
+    port = ""
+    def __init__(self):
+        self.broker_address = "110.174.81.168"
```

Comment:
store IP in configuration/ constant file?

Reply:
Inci did the pleasure of giving me the implementation of a .env file @inci90  thanks!